frontend/govuk-template/build/govuk_template.html

```diff
@@ -4,6 +4,7 @@
 <!--[if gt IE 8]><!--><html lang="{{ htmlLang }}" class="govuk-template"><!--<![endif]-->
   <head>
     <meta charset="utf-8" />
+    <meta name="robots" content="noindex">
```

Comment:
So this is for ETA and HOF is for all of our services, NHS and multiple other teams.  By implementing it this way you will impact all the teams